hydra-node/src/Hydra/Ledger/Cardano.hs

```diff
@@ -170,11 +164,7 @@ instance FromJSON Tx where
 
 instance Arbitrary Tx where
   -- TODO: shrinker!
-  arbitrary = fromLedgerTx . withoutProtocolUpdates <$> arbitrary
-   where
-    withoutProtocolUpdates tx@(Ledger.AlonzoTx body _ _ _) =
```

Comment:
Seems cool that we don't need this anymore; but what is the reason?

Reply:
Don't know.